app/src/main/java/ch/epfl/sweng/favors/database/UserGender.java

```diff
@@ -0,0 +1,46 @@
+package ch.epfl.sweng.favors.database;
+
+import android.databinding.ObservableField;
+import android.util.Log;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This class should be used when accessing user sexField
+ */
+public enum UserGender {
```

Comment:
See comment on pull request for why this class was created

Reply:
Do not really understand why you do not encapsulate the sex or gender in the User object. Because sex or gender is a field of User

Reply:
You are right the best location would be in the user. I will move it there